silhouette/test/com/mohiva/play/silhouette/impl/providers/oauth2/GitHubProviderSpec.scala

```diff
@@ -165,6 +186,11 @@ class GitHubProviderSpec extends OAuth2ProviderSpec {
     override lazy val oAuthInfo = Helper.loadJson("providers/oauth2/github.access.token.json")
 
     /**
+     * The statefulAuthInfo
```

Comment:
Such declarations must not really be documented, because no documentation will be created for test code. I nevertheless prefer the documentation, because I find it more readable, but this is only a personal preference and I can also live without the documentation. But then it should be consistent with the rest of the file. This means, if a documentation is available for a variable declaration, then it should be also available for the rest of the declarations.

So if you use the documentation, then this should be a real sentence.

```
The stateful auth info.
```